### PR TITLE
Updated Error Names

### PR DESCRIPTION
**Pull request by Nj221102**
### Description

Made error names consistent

### Checklist

- [ ] Run tests locally

### CODE REVIEW

File: lacci/lib/shoes/errors.rb [hunk 1]
```diff
@@ -16,23 +16,23 @@ class NoDrawablesFoundError < Shoes::Error; end
 
   class NoSuchStyleError < Shoes::Error; end
 
-  class NoLinkableIdError < Shoes::Error; end
+  class NoSuchLinkableIdError < Shoes::Error; end
 
   class BadLinkableIdError < Shoes::Error; end
 
-  class UnregisteredShoesEvent < Shoes::Error; end
+  class UnregisteredShoesError < Shoes::Error; end
```
Comment (noahgibbs) [inline lacci/lib/shoes/errors.rb:23]: In this case it's for an unregistered Shoes event. So we'd like to keep the word Event but add Error. Same for DoubleRegistered below.

Comment (Nj221102) [reply lacci/lib/shoes/errors.rb:23]: Ok  working on it.

Comment (Nj221102) [reply lacci/lib/shoes/errors.rb]: @noahgibbs  is it good now? or need further changes

**Review by noahgibbs — APPROVED** [review]

**Comment by noahgibbs** [comment]
Looks great! Merged.

**Pull request merged**